manifests/kots-config.yaml

```diff
@@ -3,297 +3,703 @@ apiVersion: kots.io/v1beta1
+        title: BigQuery temporary location
+        when: &when_google_workload_identy_and_connections >-
+          {{repl $cond1 := (ConfigOptionEquals "enableGoogleWorkloadIdentity" "1") -}}
+          {{repl $cond2 := (ConfigOptionEquals "enableBigqueryWorkloadIdentityConnection" "1") -}}
+          {{repl $result := and $cond1 $cond2 -}}
+          {{repl $result -}}
+        help_text: |-
+          BigQuery dataset id used for Carto Workflows to store temporary tables (i.e. `my_gcp_project.my_dataset`)
+        required: true
+        type: text
+      - name: bigqueryWorkloadIdentityConnectionBillingProject
+        title: Google billing broject
+        when: *when_google_workload_identy_and_connections
+        help_text: |-
+          GCP project to be charged with the BigQuery costs when using Workload Identity
+        required: true
+        type: text
+      - name: bigqueryWorkloadIdentityConnectionOwnerId
+        title: CARTO connection owner id
+        when: *when_google_workload_identy_and_connections
+        help_text: |-
+          Id of the Carto user who will be the owner of the bigquery connection created by workload identity (i.e. `auth0|3idsj230990sj4wsddd10`)
+        required: true
+        type: text
+
+      ## Carto Platform Credentials
+      - name: cartoPlatformGoogleSA
+        title: CARTO Platform credentials
+        when: '{{repl (ConfigOptionEquals "enableGoogleWorkloadIdentity" "0") }}'
+        help_text: |-
+          This is the Service Account  that is going to be used by the Self-Hosted environment to access CARTO platform services.
+          Please do not modify it at least that you know what you are doing.
+          Contact [Carto Support](https://docs.carto.com/carto-self-hosted/support) if you have any doubts.
+        required: true
+        type: textarea
+        default: '{{repl LicenseFieldValue "cartoPlatformDefaultSA" }}'
+      - name: cartoPlatformWorkloadIdentityEmail
+        type: textarea
+        when: '{{repl (ConfigOptionEquals "enableGoogleWorkloadIdentity" "1") }}'
+        title: |-
+          As you enable the Google Workload Identity, this is the Service Account email that is going to be used by the Self-Hosted environment to access CARTO platform services.
+          If you want to modify it, do in the Google Workload Identity section.
+        readonly: true
+        default: '{{repl (ConfigOption "googleWorkloadIdentityEmail" ) }}'
+      - name: googleServiceAccountUsed
+        title: Google Service Account used by the app
+        type: textarea
+        hidden: true
+        default: |-
+          repl{{ $workloadId := (ConfigOptionEquals "googleWorkloadIdentityStatus" "enabled") }}
+          repl{{ $result := fromJson "{}" }}
+          repl{{ if $workloadId }}
+          repl{{    $_ := set $result "sa" (ConfigOption "googleWorkloadIdentityEmail") }}
+          repl{{ else }}
+          repl{{    $_ := set $result "sa" (ConfigOption "cartoPlatformGoogleSA") }}
```

Comment:
We should extract the email from here, right? :thinking:

Reply:
what do you mean?